src/libsync/pushnotifications.cpp

```diff
@@ -36,15 +36,25 @@ void PushNotifications::reconnectToWebSocket()
 
 void PushNotifications::closeWebSocket()
 {
+    qCInfo(lcPushNotifications) << "Close websocket" << _webSocket;
+
+    _pingTimeoutTimer.stop();
+    _pingTimedOutTimer.stop();
+    _isReady = false;
+
+    // Maybe there run some reconnection attempts
+    if (_reconnectTimer) {
+        _reconnectTimer->stop();
+    }
+
     if (_webSocket) {
-        qCInfo(lcPushNotifications) << "Close websocket";
         _webSocket->close();
     }
 }
 
 void PushNotifications::onWebSocketConnected()
 {
-    qCInfo(lcPushNotifications) << "Connected to websocket";
+    qCInfo(lcPushNotifications) << "Connected to websocket" << _webSocket;
```

Comment:
@FlexW Do you really need to log the address of a pointer here? Maybe, better output something more meaningful? Like, **localAddress**, **peerAdress**, **ports**, etc?

Reply:
Ideally this would be the account URL or something similar to map to it. That's the best piece of information you would get when debugging: oh it's the websocket for *this* account which failed. I'd still log the pointer too though. That helps catch things like "why do I have two different web sockets for the same account now?"

Reply:
While debugging, the address of the websocket object was the most important information for me. But I can drop it if you want or log the url of the account too. What would you prefer?

Reply:
@FlexW URL and a pointer would be nice then. Agree with Kevin.